LondonBritishLibrary/orient/BLorient803.xml

```diff
@@ -54,72 +43,68 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">149</measure>
-                                 
-                                    
-                                    
-                                    
-                                    
-                                    <dimensions type="outer" unit="mm">
-                                        <height>311</height>
-                                        <width>283</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>12.25</height>
+                                        <width>11.125</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
                             </supportDesc>
                             <layoutDesc>
                                 <layout columns="3" writtenLines="26" cert="high">
                                 </layout>
                             </layoutDesc>
-                            
-                            
                         </objectDesc>
-                        
-                        <handDesc>
-                            
+                      <handDesc>
```

Comment:
The hand description belongs to a particular codicological unit, in this case to the first one I suppose